app/plugin_install.go

```diff
@@ -83,13 +103,12 @@ func (a *App) installPlugin(pluginFile io.ReadSeeker, replace bool) (*model.Mani
 		return nil, model.NewAppError("installPlugin", "app.plugin.mvdir.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
 
-	// Store bundle in the file store to allow access from other servers.
-	pluginFile.Seek(0, 0)
-
-	storePluginFileName := filepath.Join("./plugins", manifest.Id) + ".tar.gz"
-	if _, err := a.WriteFile(pluginFile, storePluginFileName); err != nil {
-		return nil, model.NewAppError("uploadPlugin", "app.plugin.store_bundle.app_error", nil, err.Error(), http.StatusInternalServerError)
+	// Flag plugin locally as managed by the filestore.
+	f, createErr := os.Create(filepath.Join(pluginPath, managedPluginFileName))
```

Comment:
Can we use `err` here instead of `createErr`? (and rename to `appErr` above?)

Reply:
✅